dwarf/Makefile

```diff
@@ -4,7 +4,7 @@ LIBDWARFPATH=../../libdwarf-code/
 LIBDWARFLIB=${LIBDWARFPATH}/libdwarf
 EXTRA_TARGETS+=-L$(LIBDWARFPATH)/libdwarf -ldwarf
 
-SDBPATH=../../shlr/sdb/src/
+SDBPATH=../../sdb/src/
```

Comment:
wat. are you re-shipping sdb into extras? you may use r_util which comes with sdb